src/pages/studyView/table/StructuralVariantMultiSelectionTable.tsx

```diff
@@ -0,0 +1,782 @@
+                            display: 'flex',
+                            marginRight: cellMargin,
+                        }}
+                    >
+                        {data.totalCount.toLocaleString()}
+                    </span>
+                ),
+                sortBy: (data: StructVarMultiSelectionTableRow) =>
+                    data.totalCount,
+                defaultSortDirection: 'desc' as 'desc',
+                filter: (
+                    data: StructVarMultiSelectionTableRow,
+                    filterString: string
+                ) => {
+                    return _.toString(data.totalCount).includes(filterString);
+                },
+                width: columnWidth,
+            },
+        };
+        return defaults[columnKey];
+    };
+
+    getDefaultCellMargin = (
+        columnKey: StructVarMultiSelectionTableColumnKey,
+        columnWidth: number
+    ) => {
+        const defaults: {
+            [key in StructVarMultiSelectionTableColumnKey]: number;
+        } = {
+            [StructVarMultiSelectionTableColumnKey.GENE1]: 0,
+            [StructVarMultiSelectionTableColumnKey.GENE2]: 0,
+            [StructVarMultiSelectionTableColumnKey.STRUCTVAR_SELECT]: 0,
+            [StructVarMultiSelectionTableColumnKey.NUMBER_STRUCTURAL_VARIANTS]: correctMargin(
+                getFixedHeaderNumberCellMargin(
+                    columnWidth,
+                    this.totalCountLocaleString
+                )
+            ),
+            [StructVarMultiSelectionTableColumnKey.NUMBER]: correctMargin(
+                (columnWidth -
+                    10 -
+                    (getFixedHeaderTableMaxLengthStringPixel(
+                        this.alteredCasesLocaleString
+                    ) +
+                        30)) /
+                    2
+            ),
+            [StructVarMultiSelectionTableColumnKey.FREQ]: correctMargin(
+                getFixedHeaderNumberCellMargin(
+                    columnWidth,
+                    getFrequencyStr(
+                        _.max(
+                            this.tableData.map(
+                                item =>
+                                    (item.numberOfAlteredCases! /
+                                        item.numberOfProfiledCases!) *
+                                    100
+                            )
+                        )!
+                    )
+                )
+            ),
+        };
+        return defaults[columnKey];
+    };
+
+    @computed
+    get maxNumberTotalCount() {
+        return _.max(this.tableData.map(item => item.totalCount));
+    }
+
+    @computed
+    get maxNumberAlteredCasesColumn() {
+        return _.max(this.tableData!.map(item => item.numberOfAlteredCases));
+    }
+
+    @computed
+    get totalCountLocaleString() {
+        return this.maxNumberTotalCount === undefined
+            ? ''
+            : this.maxNumberTotalCount.toLocaleString();
```

Comment:
i think we just want toString().   The use of toLocalString will make the output dependent on locale settings of browser.  I don't think we want that.  I believe we want consistent English-language formatting of numbers.

Reply:
This is code duplication with MultiSelectionTable.tsx. This specific proposal is out of scope for this PR I think. However, it highlights the need for code-deduplication.